superset-frontend/src/views/CRUD/dashboard/DashboardList.tsx

```diff
@@ -526,16 +530,26 @@ function DashboardList(props: DashboardListProps) {
           );
         }}
       </ConfirmStatusChange>
-      <ImportDashboardModal
+
+      <ImportModelsModal
+        resourceName="dashboard"
+        resourceLabel={t('dashboard')}
+        icon={<StyledIcon name="nav-dashboard" />}
+        passwordsNeededMessage={t(
+          'The passwords for the databases below are needed in order to ' +
```

Comment:
nit: i'd put this as a constant at the top

Reply:
Addressed in https://github.com/apache/incubator-superset/pull/12002